### ISSUE TITLE
npm run serve results in an error due to lazy loading syntax in the router.js file generated by vue cli

### ISSUE THREAD

**Issue by suprithwork** [vuejs/vue-cli#3344]
### Version
3.3.0

### Reproduction link
[http:/localhost:8080](http:/localhost:8080)

### Environment info
```
Vue CLI v3.3.0
? Please pick a preset: Manually select features
? Check the features needed for your project: Babel, Router, Vuex, CSS Pre-processors, Linter, Unit, E2E
? Use history mode for router? (Requires proper server setup for index fallback in production) Yes
? Pick a CSS pre-processor (PostCSS, Autoprefixer and CSS Modules are supported by default): Sass/SCSS
? Pick a linter / formatter config: Standard
? Pick additional lint features: (Press <space> to select, <a> to toggle all, <i> to invert selection)Lint on save
? Pick a unit testing solution: Mocha
? Pick a E2E testing solution: Nightwatch
? Where do you prefer placing config for Babel, PostCSS, ESLint, etc.? In dedicated config files
? Save this as a preset for future projects? No
```

### Steps to reproduce
vue create test-project

Vue CLI v3.3.0
? Please pick a preset: Manually select features
? Check the features needed for your project: Babel, Router, Vuex, CSS Pre-processors, Linter, Unit, E2E
? Use history mode for router? (Requires proper server setup for index fallback in production) Yes
? Pick a CSS pre-processor (PostCSS, Autoprefixer and CSS Modules are supported by default): Sass/SCSS
? Pick a linter / formatter config: Standard
? Pick additional lint features: (Press <space> to select, <a> to toggle all, <i> to invert selection)Lint on save
? Pick a unit testing solution: Mocha
? Pick a E2E testing solution: Nightwatch
? Where do you prefer placing config for Babel, PostCSS, ESLint, etc.? In dedicated config files
? Save this as a preset for future projects? No

npm run serve

### What is expected?
Compile successfully without compile errors

### What is actually happening?
Module parse failed: Unexpected token (19:13)
You may need an appropriate loader to handle this file type.
|     // which is lazy-loaded when the route is visited.
|     component: function component() {
|       return import(
|       /* webpackChunkName: "about" */
|       './views/About.vue');

 @ ./src/main.js 6:0-30 10:10-16
 @ multi (webpack)-dev-server/client?http://10.207.6.189:8080/sockjs-node (webpack)/hot/dev-server.js ./src/main.js

**Comment by philstavri** [comment]
Same issue, I've tried with version 3.2.1 and 3.3.0 both on Macs but with different settings than above:
```
? Please pick a preset: Manually select features
? Check the features needed for your project: Babel, Router, Vuex, CSS Pre-processors, Linter, Unit, E2E
? Use history mode for router? (Requires proper server setup for index fallback in production) Yes
? Pick a CSS pre-processor (PostCSS, Autoprefixer and CSS Modules are supported by default): Sass/SCSS
? Pick a linter / formatter config: Prettier
? Pick additional lint features: (Press <space> to select, <a> to toggle all, <i> to invert selection)Lint on save
? Pick a unit testing solution: Jest
? Pick a E2E testing solution: Cypress
? Where do you prefer placing config for Babel, PostCSS, ESLint, etc.? In dedicated config files
? Save this as a preset for future projects? No 
```

I noticed that if you set the project up with yarn (`vue create some-app -m yarn`) not npm it compiles fine... been looking into why all afternoon but I haven't found a reason to explain the issue or the difference in behaviour.

**Comment by philstavri** [comment]
Seems to be a duplicate of #3335

**Comment by sodatea** [comment]
Duplicate of #3335 and fixed by @vue/cli-service v3.3.1 released minutes ago.
